tests/foreman/cli/test_docker.py

```diff
@@ -163,7 +163,8 @@ def test_create_multiple_docker_repo(self):
 
     @tier1
     @run_only_on('sat')
-    def test_create_multiple_docker_repo_multiple_products(self):
+    def test_positive_create_multiple_docker_repos_using_multiple_products(
```

Comment:
I want to stress the fact that it is using `multiple` products. So I propose `test_positive_create_repos_using_multiple_products`